tutorials/publish/index.md

```diff
@@ -170,12 +170,12 @@ Remember that 1 AGI = 10^8 COGS, so lets set the price 1 COG.
 Then, lets set the endpoint where your `SNET Daemon` and `service` will be listening to:
 
 ```
-# snet service metadata-add-endpoints http://DAEMON_HOST:DAEMON_PORT
+# snet service metadata-add-endpoints https://DAEMON_HOST:DAEMON_PORT
```

Comment:
@arturgontijo, are you sure using `https` is correct here? I am not sure if `daemon` supports gRPC via `https` right now?

Reply:
I mean you can use `https` in endpoint but in fact gRPC endpoint be plain `http`.

Reply:
So, for now I must use `http` at service's metadata and `snet-daemon` settings?